webpack/ForemanInventoryUpload/Components/AccountList/AccountList.fixtures.js

```diff
@@ -15,7 +15,7 @@ export const API_SUCCESS_RESPONSE = {
   },
 };
 
-export const statuses = API_SUCCESS_RESPONSE;
+export const accounts = API_SUCCESS_RESPONSE;
```

Comment:
We should also update  the `API_SUCCESS_RESPONSE` with the new `label`
though this is a non-blocker, we could update that in a follow up PR.